packages/wasm/package.json

```diff
@@ -30,12 +30,154 @@
     "dist"
   ],
   "scripts": {
-    "target": "rimraf dist/$TARGET && pnpm run compile && pnpm run bindgen && pnpm run opt",
-    "compile": "cargo build --target wasm32-unknown-unknown --profile release",
-    "bindgen": "wasm-bindgen --weak-refs --target $TARGET --out-dir dist/$TARGET target/wasm32-unknown-unknown/release/wasm.wasm",
-    "opt": "wasm-opt -O4 dist/$TARGET/wasm_bg.wasm -o dist/$TARGET/wasm_bg.wasm",
-    "build": "cross-env TARGET=nodejs pnpm run target && move-file dist/nodejs/wasm.js dist/nodejs/wasm.cjs && cross-env TARGET=web pnpm run target && cpy --flat src/workerd.js dist/web && cross-env TARGET=bundler pnpm run target",
-    "test": "pw-test --mode node && pw-test"
+    "build": "export PROFILE=dev && export TARGET_DIR=debug && pnpm run build:all",
```

Comment:
`wireit` is not able to detect environment variables added by `cross-env`. Would really like to improve this part because `export` probably won't work on Windows.